From 6be334a2f21364e75ad5d62d95dfaf311ad0711b Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 1 Mar 2013 10:30:36 +0800
Subject: [PATCH 2418/2508] ENGR00252071-4: mxc_v4l2_capture: enable auto
 detect of ov5642/5640 in sabrelite

- enable the auto detect for parallel ov5642 and ov5640 in sabrelite board.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabrelite.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabrelite.c b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
index ac4b339..a0f19d2 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabrelite.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
@@ -632,7 +632,7 @@ static struct i2c_board_info mxc_i2c1_board_info[] __initdata = {
 		I2C_BOARD_INFO("mxc_hdmi_i2c", 0x50),
 	},
 	{
-		I2C_BOARD_INFO("ov5642", 0x3c),
+		I2C_BOARD_INFO("ov564x", 0x3c),
 		.platform_data = (void *)&camera_data,
 	},
 };
-- 
1.7.7.4

